pkg/apis/apps/v1alpha1/defaults.go

```diff
@@ -36,4 +36,8 @@ func SetDefaultsBroadcastJob(job *BroadcastJob) {
 		parallelismIntStr := intstr.FromInt(int(parallelism))
 		job.Spec.Parallelism = &parallelismIntStr
 	}
+
+	if job.Spec.FailurePolicy.Type == "" {
```

Comment:
SetDefaultBroadcastJob func should be removed from this file (can you do this in this change? Thanks)

Please move the initialization code to 

https://github.com/openkruise/kruise/blob/e0e67c8faaf3d5cd5afc50028374caf6af859460/pkg/webhook/default_server/broadcastjob/mutating/broadcastjob_create_update_handler.go#L57